src/EnergyPlus/ElectricPowerServiceManager.cc

```diff
@@ -116,6 +117,7 @@ namespace EnergyPlus {
 	)
 	{
 		if ( getInputFlag_ ) {
+			if ( !DataErrorTracking::AskForPlantCheckOnAbort ) return;
```

Comment:
First call to this function will not execute. See line 1593 of HVACManager.cc where manageElectricPowerService is called, then ManagePlantLoops is called, then AskForPlantCheckOnAbort = true and finally manageElectricPowerService is called again. This change means that only the first call of the simulation is avoided.